manifests/mod/ssl.pp

```diff
@@ -95,7 +95,7 @@
   Optional[Stdlib::Absolutepath] $ssl_cert                  = undef,
   Optional[Stdlib::Absolutepath] $ssl_key                   = undef,
   Optional[Stdlib::Absolutepath] $ssl_ca                    = undef,
-  String $ssl_cipher                                        = $apache::params::ssl_cipher,
+  Variant[String, Array, Hash] $ssl_cipher                  = $apache::params::ssl_cipher,
```

Comment:
Why `Array`? I don't see code in the template to handle that. And can we further tighten `Hash` to `Hash[String[1], String[1]]`?

Reply:
Thanks for your comments, I wasn't aware that we can even define variable that level in puppet.

Reply:
Puppet data types are surprisingly powerful and I like them a lot.